packages/consensus/fcob/consensusmechanism_test.go

```diff
@@ -219,7 +219,6 @@ func TestOpinionFormer_Scenario2(t *testing.T) {
 	testTangle.ConsensusManager.Events.MessageOpinionFormed.Attach(events.NewClosure(func(messageID tangle.MessageID) {
 		t.Logf("MessageOpinionFormed for %s", messageID)
 
-		assert.True(t, testTangle.ConsensusManager.MessageEligible(messageID))
```

Comment:
This has been deleted since consensus mechanism is not in charge of eligibility and it has no place in the unit test